server/tests/api_v4/authenticated/test_invite_greeter_start_greeting_attempt.py

```diff
@@ -1,29 +1,38 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
-import pytest
 
-from parsec._parsec import GreetingAttemptID, InvitationToken, authenticated_cmds
+from parsec._parsec import (
+    GreetingAttemptID,
+    InvitationToken,
+    authenticated_cmds,
+)
 from tests.common import CoolorgRpcClients, HttpCommonErrorsTester
 
 Response = authenticated_cmds.v4.invite_greeter_start_greeting_attempt.Rep | None
 
 
-# TODO: Remove once PostgreSQL is supported
-@pytest.fixture(autouse=True)
-def _skip_if_postgresql(skip_if_postgresql: None) -> None:  # type: ignore
-    pass
-
-
 async def test_authenticated_invite_greeter_start_greeting_attempt_ok(
     coolorg: CoolorgRpcClients,
 ) -> None:
+    # This is a scenario where the greeter starts first.
```

Comment:
Should we add a scenario where the claimer starts first ?

(the test that assert that all tests are here doesn't checks if there are more tests rigth?)

Reply:
Sorry it's an artefact from a test I moved in `test_greeting_attempt.py` it should be removed